pandas/tests/test_groupby.py

```diff
@@ -4594,9 +4594,6 @@ def test_frame_groupby_plot_boxplot(self):
         _skip_if_mpl_not_installed()
 
         import matplotlib.pyplot as plt
-        import matplotlib as mpl
```

Comment:
Ah I didn't know you had already a `TestDataFrameGroupByPlots` in test_graphics. Yep, certainly OK then to move the tests there.

But is seems  a lot of the tests in `TestDataFrameGroupByPlots` don't really belong there (as it are just Series.plot tests)